crates/levm/src/transaction.rs

```diff
@@ -0,0 +1,70 @@
+use bytes::Bytes;
+use ethereum_types::{Address, H256, U256};
+
+type AccessList = Vec<(Address, Vec<U256>)>;
+// type VersionedHash = H32;
+
+/// Transaction destination.
+#[derive(Clone, Debug)]
+pub enum TransactTo {
+    /// Simple call to an address.
+    Call(Address),
+    /// Contract creation.
+    Create,
+}
+
+/// The transaction environment.
+#[derive(Clone, Debug)]
+pub struct TxEnv {
+    /// Caller aka Author aka transaction signer.
+    pub caller: Address,
```

Comment:
```suggestion
    pub msg_sender: Address,
```
Let's rename this to msg_sender to keep the same naming everywhere

Reply:
Ok!